tests/CMakeLists.txt

```diff
@@ -10,43 +10,19 @@ add_library(test_common INTERFACE)
 target_link_libraries(
     test_common
     INTERFACE
-        umd_device
+        umd::device
+        umd::Firmware
         gtest_main
         gtest
         pthread
-        fmt
+        fmt::fmt-header-only
 )
 target_include_directories(
     test_common
     INTERFACE
         ${PROJECT_SOURCE_DIR}
         ${CMAKE_CURRENT_SOURCE_DIR}
 )
-if($ENV{ARCH_NAME} STREQUAL "grayskull")
-    message(STATUS "UMD: Building for Grayskull")
-    target_include_directories(
-        test_common
-        INTERFACE
-            ${PROJECT_SOURCE_DIR}/device/grayskull
```

Comment:
Where did this (and for other archs) include go?
I see where the other one, src/firmware/riscv, landed...

Reply:
I took it out and nothing broke 🤷  so I figured it was dead code.

Reply:
Maybe those files are directly included and the include paths aren’t necessary. Or they could be dead code. I’ll take a close look in a bit.

Reply:
Should I hold off merging this PR?  Or okay to merge since everything is green?  (Or is there something to manually trigger?)